tests/package/test_models.py

```diff
@@ -444,6 +444,12 @@ def test_should_update_data_package_if_preexists(self):
                    Package.version == 'tag_one').one()
         self.assertEqual(latest_data.readme, tagged_data.readme)
 
+    def test_should_return_true_if_package_already_exists(self):
```

Comment:
I think this test case could be better named -- *what* should return true if package exists ...?

Given it is part of a general `TestPackage` test case, i suggest calling it simply:

```
test_is_package_exists
```